### ISSUE TITLE
Recommended Content

### ISSUE THREAD

**Issue by 6102bitcoin** [bitcoin-only/bitcoin-only-rebranding#5]
I am trying to figure out the best way to have recommended entries. 

I have tried the following.
1) Highlighting the rows
2) Highlighting the rows & a row break
3) Two tables

Would appreciate suggestions. 

1) Highlighting the rows 
![image](https://user-images.githubusercontent.com/42300012/93941665-d3aaeb80-fd26-11ea-875c-8a1e87c43752.png)

2) Highlighting the rows & a row break 
![image](https://user-images.githubusercontent.com/42300012/93941793-110f7900-fd27-11ea-982b-003e5afd55ca.png)

3) Two Tables (Could be with or without Highlighting)
![image](https://user-images.githubusercontent.com/42300012/93941973-651a5d80-fd27-11ea-9952-3166879b83a2.png)

**Comment by 6102bitcoin** [comment]
3) Two Tables (Could be with or without Highlighting)
![image](https://user-images.githubusercontent.com/42300012/93942031-8418ef80-fd27-11ea-8be4-5e4cada6d83e.png)

**Comment by wiz** [comment]
I think 3 is best, separating them into their own featured table at the top

**Comment by wiz** [comment]
@pedromvpg hey Pedro, you're a leet designer, what do you think? :grin:

**Comment by pedromvpg** [comment]
Would consider just hightailing it with an icon along with the word "recommended"

![image](https://user-images.githubusercontent.com/807505/93976256-b9810580-fd46-11ea-832f-38a6b620445d.png)

**Comment by 6102bitcoin** [comment]
Nice idea. Might need to drop to just the icon on small screens. 

I'm interested in the rest of the screenshot - links aren't underlined & headers are grey - were these design suggestions also?

**Comment by pedromvpg** [comment]
Yes, it was part of the design proposal but got lost in the commit. 
Will add later if you like it.
